### DISCUSSION TITLE
Is there any syntax can match values which containing specific words?

### DISCUSSION THREAD

**Discussion by 1000Fear** [mpv-player/mpv#14821] [Q&A] [answered]
Just like `sub-auto` and `audio-file-auto` = `fuzzy` to make them load all file containing the media filename.

I want the profile to take effect based on the media title.
For example, here are the media titles I have:

> Fargo S01E01 The Crocodile's Dilemma
Fargo S01E02 The Rooster Prince
Fargo S01E03 A Muddy Road

I want all files with titles containing 'Fargo' to be matched, how should I fill in the value of `profile-cond`?
`profile-cond=get("match syntax") == "Fargo"`

P.S. it's **media title** not **file name**

**Comment by guidocella** [comment, marked as answer]
`media_title:find('Fargo')`

**Reply by 1000Fear** [reply]
Thank you! learned new knowledge again :)

**Reply by hooke007** [reply]
It actually was recorded in manual.
https://mpv.io/manual/master/#conditional-auto-profiles
To check `Example`

**Reply by 1000Fear** [reply]
@hooke007 Thank you for pointing it out! I didn't realize that 'find' could be used this way. I still have a lot to learn... :)
